diff --git a/dist/index.cjs b/dist/index.cjs
index 2ad268b7fbc9568564e47de47269d6a1f2ea070e..eb8d12d1da4d6f266ba84804ff34c63ce8aeb158 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -38,21 +38,11 @@ function presetDaisyui(option) {
             decl.prop = decl.prop.replaceAll(/(var\s*\(\s*)?--(?:tw-)+([-\w]+)?/g, "$1--un-$2");
             decl.value = decl.value.replaceAll(/(var\s*\(\s*)?--(?:tw-)+([-\w]+)?/g, "$1--un-$2");
           }
-        },
-        {
-          postcssPlugin: "format-css",
-          Declaration: (decl) => {
-            if (decl.value?.includes("hsl"))
-              decl.value = decl.value.replaceAll(/\)\s?\/\s?/g, "), ");
-            decl.value = decl.value.replaceAll(/([\d.]+)\s+([\d.%]+)\s+([\d.%]+)\s?$/g, "$1, $2, $3");
-          }
         }
       ]
     }),
     name: "unocss-preset-daisyui",
-    theme: {
-      colors: daisy__default.config?.theme?.extend?.colors
-    }
+    theme: daisy__default.config?.theme?.extend
   };
 }
 
diff --git a/dist/index.mjs b/dist/index.mjs
index ab023d082c0049f13a5b1941d0b9cc2733069af8..6d988d697538211fdee84b901aa3c5f921b645ed 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -30,21 +30,11 @@ function presetDaisyui(option) {
             decl.prop = decl.prop.replaceAll(/(var\s*\(\s*)?--(?:tw-)+([-\w]+)?/g, "$1--un-$2");
             decl.value = decl.value.replaceAll(/(var\s*\(\s*)?--(?:tw-)+([-\w]+)?/g, "$1--un-$2");
           }
-        },
-        {
-          postcssPlugin: "format-css",
-          Declaration: (decl) => {
-            if (decl.value?.includes("hsl"))
-              decl.value = decl.value.replaceAll(/\)\s?\/\s?/g, "), ");
-            decl.value = decl.value.replaceAll(/([\d.]+)\s+([\d.%]+)\s+([\d.%]+)\s?$/g, "$1, $2, $3");
-          }
         }
       ]
     }),
     name: "unocss-preset-daisyui",
-    theme: {
-      colors: daisy.config?.theme?.extend?.colors
-    }
+    theme: daisy.config?.theme?.extend
   };
 }
 
diff --git a/package.json b/package.json
index 8f8260595951b1cbbd2a4a27a15f8bb04009aa3c..a40d7e8dc6b8cea22639400b46fe8b4c54c4a3bc 100644
--- a/package.json
+++ b/package.json
@@ -3,8 +3,20 @@
   "version": "0.0.2",
   "author": "",
   "license": "MIT",
-  "description": "",
-  "keywords": [],
+  "description": "UnoCSS preset for daisyUI",
+  "homepage": "https://github.com/Super-Lollipop/unocss-preset-daisyui",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Super-Lollipop/unocss-preset-daisyui"
+  },
+  "bugs": {
+    "url": "https://github.com/Super-Lollipop/unocss-preset-daisyui/issues"
+  },
+  "keywords": [
+    "unocss",
+    "daisyui",
+    "unocss-preset"
+  ],
   "main": "dist/index.cjs",
   "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
@@ -18,27 +30,27 @@
       "require": "./dist/index.cjs"
     }
   },
+  "scripts": {
+    "build": "unbuild",
+    "stub": "unbuild --stub",
+    "play": "npm -C playground run dev",
+    "release": "bumpp && pnpm build && pnpm publish"
+  },
   "devDependencies": {
-    "@antfu/eslint-config": "^0.41.0",
-    "@types/node": "^20.5.9",
-    "@types/postcss-js": "^4.0.1",
+    "@antfu/eslint-config": "^2.3.4",
+    "@types/node": "^20.10.3",
+    "@types/postcss-js": "^4.0.4",
     "bumpp": "^9.2.0",
-    "eslint": "^8.48.0",
-    "typescript": "^5.2.2",
+    "eslint": "^8.55.0",
+    "typescript": "^5.3.2",
     "unbuild": "^2.0.0"
   },
   "peerDependencies": {
-    "daisyui": ">=3.0.0",
-    "unocss": ">=0.53.0"
+    "daisyui": ">=4.0.0",
+    "unocss": ">=0.58.0"
   },
   "dependencies": {
-    "autoprefixer": "^10.4.15",
+    "autoprefixer": "^10.4.16",
     "unocss-preset-component": "^0.0.4"
-  },
-  "scripts": {
-    "build": "unbuild",
-    "stub": "unbuild --stub",
-    "play": "npm -C playground run dev",
-    "release": "bumpp && pnpm build && pnpm publish"
   }
 }
